lib/request-handler.js

```diff
@@ -251,7 +251,7 @@ RequestHandler.prototype.prepareMultiple = function (queries, callbacksArray, op
   Error.captureStackTrace(this.stackContainer = {});
   var self = this;
   this.retryHandler = function () {
-    //Use a custom retryHandler
+    // Use a custom retryHandler
     self.prepareMultiple(queries, callbacksArray, options, callback);
```

Comment:
Ah of course, that makes sense, I didn't consider at the time that it would use the query plan to select hosts. :+1: